packages/forma-36-react-components/src/components/CheckboxField/README.mdx

```diff
@@ -0,0 +1,64 @@
+# CheckboxField
+
+CheckboxField is a component that gives an possiblity to easy setup proper checkbox with its label. Component also contains an options that allows setting helper text displayed below the label. You might see checkboxes in forms, they might be also used for filtering data on the page or terms and conditions. Everywhere where user have multiple choice you might want to use this component.
```

Comment:
@burakukula What do you think about writing the intro paragraph in the form of **when to use** the component?
(instead of describing the component parts and where users might find it)

So, something like:

_Use a CheckboxField to give users a way to select items from a list, when they could select zero, one or more items._

Reply:
- without mentioning variants (helper text in this case)

Reply:
- focusing on **when** to use the component, not where, I would omit the part with forms, but maybe keep "e.g. when users need to agree to Terms and conditions."

Reply:
@domarku I made some changes, I still think forms are a valid example of usage. Please have a look if this is close to what you have in mind?